addons/mass_mailing_crm/models/mass_mailing.py

```diff
@@ -31,30 +34,35 @@ def _compute_crm_opportunities_count(self):
             if mass_mailing.crm_lead_activated:
                 mass_mailing.crm_opportunities_count = 0
             else:
-                mass_mailing.crm_opportunities_count = self.env['crm.lead'].search_count(self._get_crm_utm_domain())
+                mass_mailing.crm_opportunities_count = self.env['crm.lead'] \
```

Comment:
As both fields seem to have the same meaning I would remove one of them. No need to have twice the same code.

Reply:
As discussed, we have to keep 2 fields since they have different text AND they redirect to different views.

But I have merged the 2 "compute" methods into one so it's a little bit cleaner now at least.